### PR TITLE
[Service Connector] `az webapp connection create`: add parameter to be compatible with CLI core

### PR DESCRIPTION
**Pull request by xfz11**
---

This checklist is used to make sure that common guidelines for a pull request are followed.
Fix https://github.com/Azure/azure-cli/issues/28865
### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Service Connector

**Comment by yonzhan** [comment]
Please fix CI issues.

File: src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_params.py [hunk 1]
```diff
@@ -75,9 +77,11 @@ def load_arguments(self, _):
                 add_source_resource_block(c, source, enable_id=False)
                 add_target_resource_block(c, target)
                 add_auth_block(c, source, target)
+                add_configuration_store_argument(c)
                 add_new_addon_argument(c, source, target)
                 add_secret_store_argument(c)
```
Comment (wchigit) [inline src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_params.py:82]: Could you please help add a new parameter to this function to align with CLI?
add_secret_store_argument(c, source)

**Review by wchigit — APPROVED** [review]

File: src/serviceconnector-passwordless/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+1.1.0
```
Comment (AllyW) [inline src/serviceconnector-passwordless/HISTORY.rst:5]: Deleted args `key_vault_id` is a breaking change. Major version number should be increased to be 2.0.0.

For more info, please refer to the extension versioning doc: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#versioning-scheme

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**